templates/empty-skeleton/daml/README.md

```diff
@@ -0,0 +1,10 @@
+This is the source directory for your DAML files. Each file will start with a module header like
```

Comment:
Maybe a `.gitkeep` file would be nicer? It seems a bit odd to have a README here.

Reply:
added the .gitkeep file, removed the readme.